README.md

```diff
@@ -184,8 +206,8 @@ npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standa
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
-
-- **Add, edit and display category**
+- Implement some UX improvements: include transitions and/or animations, etc.
+- Implement additional pages in the website (each with a route in the SPA): about me, references, etc.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Under the "Acknowledgements" section please give  appropriate credit to the designer